system/gatekeeper/tests/gatekeeper-policies/fixtures/restrict-monsoon3-deployments/configmap-allowed.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bogus-pattern-for-testing
+  namespace: monsoon3
+data:
+  test-key: "test-value"
+  another-key: "another-value"
```

Comment:
I don't want for this policy to match on everything. That will require an excessive level of detail in the object name regexes. We should only match on Helm releases. Please review the existing policy GkOwnerInfoOnHelmReleases for how to match Helm releases, specifically:

- the match expression on the constraint: https://github.com/sapcc/helm-charts/blob/master/system/gatekeeper-config/templates/constraint-owner-info-on-helm-releases.yaml#L12
- the tests for that policy that construct Helm release objects out of a specification file

Helm release objects are always `iro.kind == "Secret"` and are named according to the pattern `sh.helm.release.v1.$NAME.v$VERSION`, so you can match the name to extract the release name without having to parse the full release manifest.